sequencer/src/persistence/sql.rs

```diff
@@ -486,9 +488,9 @@ impl Persistence {
             if hash.is_none() {
                 let view: i64 = row.try_get("view")?;
                 let data: Vec<u8> = row.try_get("data")?;
-                let proposal: Proposal<SeqTypes, QuorumProposal<SeqTypes>> =
+                let proposal: Proposal<SeqTypes, QuorumProposal2<SeqTypes>> =
```

Comment:
@imabdulbasit sorry, why this change? Aren't we storing V1 types?

Reply:
tests were failing, and this change made them pass, but this needs to be reverted. 
reverted and fixed test_quorum_proposals_leaf_hash_migration()

https://github.com/EspressoSystems/espresso-sequencer/pull/2344/commits/a84d23244ee926618a464d2428ef20fec7f2e838